web/_includes/consolidation.html

```diff
@@ -0,0 +1,42 @@
+{% if include.schema_data.consolidation %}
+  {% assign consolidation = include.schema_data.consolidation %}
+
+  <h2>Jeu de données consolidé</h2>
+
+  <p>
+    Une consolidation de données est effectuée à partir de ce schéma. Vous pouvez retrouver le jeu de données résultant de cette consolidation sur data.gouv.fr
+  </p>
+
+  <div data-udata-dataset="{{ consolidation.dataset_id }}"></div>
+
+  <div class="js-consolidation-details consolidation-details"></div>
+
+  <script>
+    const datasetId = document.querySelector('[data-udata-dataset]').getAttribute('data-udata-dataset')
+
+    const maybePluralize = (count, noun, suffix = 's') => `${count} ${noun}${count !== 1 ? suffix : ''}`
```

Comment:
is this code transpiled? if not, are the string templates wildly supported enough across browsers?